packages/document/docs/zh/plugins/list/_meta.json

```diff
@@ -16,6 +16,7 @@
   "plugin-image-compress",
   "plugin-mdx",
   "plugin-node-polyfill",
+  "plugin-lightningcss",
```

Comment:
the sidebar is disappeared...

https://deploy-preview-1741--rsbuild.netlify.app/plugins/list/plugin-lightningcss

![image](https://github.com/web-infra-dev/rsbuild/assets/7237365/4ac2b848-6147-46b8-b68a-69a80716bbd4)